### ISSUE TITLE
Minimal features needed for citation management

### ISSUE THREAD

**Issue by burtonator** [burtonator/polar-bookshelf#1116]
- extended schema including author, journal, publication date and the ability to filter and sort over these fields.

- import+export from bibtex 

- Ability to search using APIs like arxiv and unpaywall , add a document, and add it to a Polar including the PDF and keep the metadata correctly.

- tag a set of references and then export them using a CSL bibliographic style.

**Comment by bepolymathe** [comment]
Maybe just an a connexion with zotero ?

**Comment by tbsexton** [comment]
@burtonator @bepolymathe  I completely agree. This is a win-win -- zotero really does not implement high-level PDF annotation, but is a fantastic ref. manager. Since Linux users (myself) don't have access to apps like [Highlights](https://highlightsapp.net/) that do integrate with *some kind* of citation manager, Polar has a chance to set up as a great tool for academics specifically. 

My use case really is adding tagging/annotation/reading QoL to my zotero workflow, not switching over to polar for citations. Doesn't Zotero store pure-PDFs in its database, anyway? Could be an easy feature, potentially.
